state-synchronizer/src/tests/integration_tests.rs

```diff
@@ -206,7 +206,7 @@ impl SynchronizerEnv {
     }
 
     fn new(num_peers: usize) -> Self {
-        set_simple_logger("state-sync");
+        Logger::new().filter("state-sync", Level::Trace).init();
```

Comment:
Remove the import of `Logger` and call it with `::libra_logger::Logger...` like everywhere else.